tilings/algorithms/minimal_gridded_perms.py

```diff
@@ -485,9 +486,14 @@ def _process_work_packet(
             # if gp was one of the initial_gps that satisfied obs/reqs, but
             # we weren't sure at the time if it was minimal, then now is the
             # time to check and yield
-            if qpacket.gp in initial_gps_to_auto_yield:
+            if curr_len != len(qpacket.gp):
+                curr_len = len(qpacket.gp)
+                for gp in initial_gps_to_auto_yield[curr_len]:
+                    yielded.add(gp)
+                    yield gp
```

Comment:
you need to double check gp is minimal here
```python
if not yielded_subgridded_perm(gp):
    yielded.add(gp)
    yield gp
```

Reply:
True just fixed it